### ISSUE TITLE
Translate the game

### ISSUE THREAD

**Issue by KERRIGANRAYNOR** [leszekd25/spellforce_data_editor#13]
How can i change any dialog, name etc? i want translate the game

**Comment by leszekd25** [comment]
Best shot would be to open gamedata, go to category 15. Text data, and start translating. Category 15 elements have up to 5 defined language entries, so you need to choose one that's used by your version of the game (typically 1) and modify that.
Note that doing this will not let you use any of the big mods posted on nexusmods (like spellforce reworked, reimagined etc.)

**Comment by KERRIGANRAYNOR** [comment]
Thank you, I want to translate the spellforce reworked mod from English to Spanish. there are many bugs when I use the Spanish version with this mod.

**Comment by leszekd25** [comment]
Yep, the Reworked mod only bothered to add text in english language (which is understandable, given that actually translating all the new text entries is a lot of work)

**Comment by KERRIGANRAYNOR** [comment]
The problem is that the mod only added 2 language ID (0 deutsche and 1 English) if the mod had added the language ID 2, 3 (Spanish) and 4 we could see the new text (Although it was parts in English and parts in Spanish of the original game). Btw thank you for this tool, its awesome. <3 .
